components/automate-ui/src/app/pages/+compliance/+scanner/containers/job-scans-list/job-scans-list.component.scss

```diff
@@ -5,12 +5,19 @@
 }
 
 .jobs-list-table {
-  .new-row {
-    font-size: 14px;
-  }
+  chef-table-cell {
+    display: flex;
+    justify-content: center;
+
+    &:last-child {
+      flex-flow: row nowrap;
+      justify-content: flex-end;
+      align-items: center;
 
-  chef-td:last-child {
```

Comment:
As close as I could mark it: Need to delete the similar line from src/app/pages/+compliance/+scanner/containers/scanner/scanner.component.scss::L45
```
::ng-deep chef-td:last-child {
```